less/transferwise.less

```diff
@@ -36,6 +36,27 @@ address {
 }
 .footer {
     background-color: @gray-base;
+
+	.icon {
+		font-size: 50px !important;
+		margin: 0 5px 0 5px;
+	}
+
+	h5 {
+		a {
+			color: inherit !important;
+			font-weight: inherit;
+		}
+	}
+
+	a {
+		text-decoration: none;
+		font-weight: normal;
```

Comment:
Hmm just on not having 20 links underlined in the footer... Not sure if there are official designs for that...